CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 ## Unreleased
 
+* (mvz) Add workaround for issues/653
```

Comment:
I'd prefer it to have a description here. If we just use issue numbers we can throw away the changelog and use git again :)